DOMAINS_LINK.txt

```diff
@@ -1 +1,4 @@
 https://github.com/GSA/data/tree/gh-pages/dotgov-domains
+
+Current Federal Data from:
+https://raw.githubusercontent.com/GSA/data/gh-pages/dotgov-domains/current-federal.csv
```

Comment:
Are these just notes on where the data is being parsed from? If yes - I wonder if we should just put them in the README.